docs/src/main/sphinx/develop/supporting-merge.rst

```diff
@@ -3,24 +3,25 @@ Supporting ``MERGE``
 ====================
 
 The Trino engine provides APIs to support row-level SQL ``MERGE``.
-To implement ``MERGE``, a connector must provide an implementation
-of ``ConnectorMergeSink``, which is typically layered on top of a
-``ConnectorPageSink``, and define ``ConnectorMetadata``
-methods to get a "rowId" column handle; get the row change paradigm;
-and to start and complete the ``MERGE`` operation.
+To implement ``MERGE``, a connector must:
```

Comment:
It's weird to split a sentence with bullets. I don't see that in the [https://developers.google.com/style/lists](style guide), and would be surprised if it's there, since that wouldn't be proper English.

* The colon should be preceded by a complete sentence (the "introductory sentence").
* Make each of the items a capitalized sentence that ends with a period.

Reply:
Thanks, I forgot about using full sentences here. The style guide suggests avoiding semicolons and I noticed the list here has two levels - the second point also lists multiple items. I thought using bullet points will make it easier to read by avoiding nesting.